README.adoc

```diff
@@ -103,6 +105,56 @@ class MyComponent @Inject() (mailerClient: MailerClient) {
 }
 ```
 
+#### With compile-time injection
+
+```scala
+import play.api.libs.mailer._
+import javax.inject.Inject
+import java.io.File
+import org.apache.commons.mail.EmailAttachment
+
+class MyComponent(mailerClient: MailerClient) {
+
+  def sendEmail {
```

Comment:
I think it's quite useful to have a method that use the application.conf settings and another one that uses programmatic settings